yargs/yargs-tests.ts

```diff
@@ -12,539 +12,539 @@ const stringVal = 'string';
 
 // With yargs, the options be just a hash!
 function xup() {
-	var argv = yargs.argv;
+    var argv = yargs.argv;
```

Comment:
we should include `.editorconfig`
such as https://github.com/DefinitelyTyped/DefinitelyTyped/blob/36d40a63a05b5cb3413737fbdf7c9a2a26f211fd/baconjs/.editorconfig